src/cargo/ops/cargo_compile/mod.rs

```diff
@@ -493,14 +497,36 @@ pub fn create_bcx<'a, 'gctx>(
                     continue;
                 };
 
-                warn_on_unexpected_cfgs(gctx, check_cfg, &cfg_expr, None, "")?;
+                warn_on_unexpected_cfgs(
+                    gctx,
+                    check_cfg,
+                    &cfg_expr,
+                    LintLevel::Warn,
+                    &mut error_count,
+                    None,
+                    "",
+                )?;
```

Comment:
The fact that this isn't subject to lint config calls into question whether we should warn.  We are very cautious of adding warnings that cannot be suppressed in some way.

Does this correspond with config?

Reply:
Yeah, I don't like this either, but I don't know to which config it should depend.

Maybe we could let this a future possibility until we figure out the anwser.

Reply:
Another option is we keep it in the unstable flag and not stabilize it with the rest until we figure out our story for linting of config.